tests/lib/rules/no-else-return.js

```diff
@@ -97,6 +97,18 @@ ruleTester.run("no-else-return", rule, {
             errors: [{ message: "Unnecessary 'else' after 'return'.", type: "BlockStatement" }]
         },
         {
+            code: "function foo9a() {if (x) { return true; } else if (y) { return true; } else { notAReturn(); } }",
+            output: "function foo9a() {if (x) { return true; } if (y) { return true; } else { notAReturn(); } }",
+            options: [{ allowElseIf: false }],
+            errors: [{ message: "Unnecessary 'else' after 'return'.", type: "IfStatement" }]
+        },
+        {
+            code: "function foo9b() {if (x) { return true; } if (y) { return true; } else { notAReturn(); } }",
+            output: "function foo9b() {if (x) { return true; } if (y) { return true; }  notAReturn();  }",
+            options: [{ allowElseIf: false }],
+            errors: [{ message: "Unnecessary 'else' after 'return'.", type: "BlockStatement" }]
```

Comment:
Can you add some tests with `allowElseIf: true`?

Reply:
they're all allowElseIf: true by default....

Reply:
I'm aware of that. I'm asking if you can add a few tests which explicitly use `allowElseIf: true`, to make sure the case where that option is explicitly provided is handled correctly.

Reply:
up I've added some now @not-an-aardvark